internal/handler/sjson/sjson_test.go

```diff
@@ -202,21 +204,28 @@ func addRecordedFuzzDocs(f *testing.F, needDocument, needSchema bool) int {
 
 		switch b := rec.Body.(type) {
 		case *wire.OpMsg:
-			doc, err := b.Document()
+			var raw wirebson.AnyDocument
+			raw, err = b.RawDocument()
 			require.NoError(f, err)
+
+			var doc *types.Document
+			doc, err = bson.TypesDocument(raw)
+			require.NoError(f, err)
+
 			docs = append(docs, doc)
 
 		case *wire.OpQuery:
-			if doc := b.Query(); doc != nil {
-				docs = append(docs, doc)
-			}
+			if raw := b.Query(); raw != nil {
+				var doc *types.Document
+				doc, err = bson.TypesDocument(raw)
+				require.NoError(f, err)
 
-			if doc := b.ReturnFieldsSelector(); doc != nil {
```

Comment:
At the moment, `ReturnFieldsSelector` is not exposed. Do we want expose it so we can add it to fuzzing?

Reply:
Created a separate PR for removing all fuzzing related stuff :+1: